tests/tests.cpp

```diff
@@ -43,6 +43,7 @@
 #include <string>
 
 #include "libzippp.h"
+#include <zip.h> // ZIP_CM_DEFLATE
```

Comment:
Actually the goal is to avoid import of `<zip.h>` all together. You should make some `#define` in libzippp.h that are mapped to the ones of libzip.